ouroboros-consensus/src/Ouroboros/Consensus/Node/NetworkProtocolVersion.hs

```diff
@@ -74,6 +67,17 @@ class ( Show (NodeToNodeVersion   blk)
     => Proxy blk -> NodeToClientVersion blk
   mostRecentNodeToClientVersion _ = ()
 
+class HasNetworkProtocolVersion blk => TranslateNetworkProtocolVersion blk where
+  -- | Translate to network-layer type
+  nodeToNodeProtocolVersion
+    :: Proxy blk -> NodeToNodeVersion blk -> N.NodeToNodeVersion
+
+  -- | Translate to network-layer type
+  nodeToClientProtocolVersion
+    :: Proxy blk -> NodeToClientVersion blk -> N.NodeToClientVersion
+
+  -- Defaults
+
```

Comment:
Should we make `N.NodeToClientV_2` the default, now that the `LocalStateQuery` protocol has been enabled?

Reply:
:shrug:

Reply:
Defaults are now gone, and V2 is now the enabled for the mock ledger.